ToolsTest/Volodymyr/MSTest.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+using Assert = Microsoft.VisualStudio.TestTools.UnitTesting.Assert;
+
+using Tools;
+using Tools.Models;
+
+namespace ToolsTest.Volodymyr
+{
+    [TestClass]
+    public class MSTest
+    {
+        //TryReplaceText GetCapitalTitle HasText
+        [TestMethod]
+        public void TestGetCapitalTitle()
+        {
+            //arrange
+            Article article = new Article { Title = "same article" };
+
+            //act
+            string result = article.GetCapitalTitle();
+
+            // Assert
+            Assert.AreEqual(" SAME ARTICLE", result);
```

Comment:
Here the test fails because of an extra space. Remove the space before the `SAME` word.
```suggestion
            Assert.AreEqual("SAME ARTICLE", result);
```